src/error.test.ts

```diff
@@ -8,6 +8,17 @@ describe('Error', () => {
         jest.resetAllMocks();
     });
 
+    it('runs the default error handler correctly if no handler has been set', () => {
+        /* eslint-disable no-console */
+        console.error = jest.fn();
```

Comment:
Will `console.error` be reset to it's true/original state following this test? If not I'd suggest we follow the pattern we've used else where for global objects/functions:
```js
global.console = {error: jest.fn()};
expect(console.error).toBeCalled();
global.console.mockClear();
delete global.console;
```

Reply:
👍